web/wp-content/themes/dctx-v2/index.php

```diff
@@ -0,0 +1,55 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-wcngqf.php, but found phpcs-scan-WCNgqF.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).